Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: test lint autoformat run ftest install release docker-build docker-run docker-push
 
-PYTHON=python3.10
+PYTHON=python
```

Comment:
Not sure whether having it `python` or `python3` here is better, open to discussion here

Reply:
Do we even need this anymore? Now we should just be providing guidance to make sure that you you've installed a compatible system python version, or set up `pyenv` to enforce a compatible version. Right?